internal/pkg/application/functions/registry.go

```diff
@@ -63,7 +63,7 @@ func NewRegistry(ctx context.Context, input io.Reader, storage database.Storage)
 			} else if f.Type == levels.FunctionTypeName {
 				levelConfig := ""
 				if tokenCount > 5 {
-					levelConfig = tokens[5]
+					levelConfig = tokens[6]
```

Comment:
What is the minimum amount of tokens you need to be able to safely dereference the sixth element?